crypto/evp/evp_enc.c

```diff
@@ -294,8 +294,9 @@ int is_partially_overlapping(const void *ptr1, const void *ptr2, int len)
     return overlapped;
 }
 
-int EVP_EncryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
-                      const unsigned char *in, int inl)
+static int evp_EncryptDecryptUpdate(EVP_CIPHER_CTX *ctx,
+                                    unsigned char *out, int *outl,
+                                    const unsigned char *in, int inl)
```

Comment:
Did not check for precedents, but wouldn't an all-lowercase name be more appropriate for an internal function? Something like `[evp_]encrypt_decrypt_update()` ?

Reply:
I dunno, seriously...  lower case prefix, but then follow the style of the file, was my line of thinking.

Reply:
Probably you're right. The style of the file (mixed_underscore_withCamelCase) is the problem ;-)